src/include/btmem.h

```diff
@@ -1099,7 +1102,11 @@ struct __wt_update {
  * WT_UPDATE_SIZE is the expected structure size excluding the payload data -- we verify the build
  * to ensure the compiler hasn't inserted padding.
  */
+#ifdef HAVE_DIAGNOSTIC
+#define WT_UPDATE_SIZE 48
```

Comment:
I'm terrible at this game, but it seems odd that adding 8 bytes increased the data structure size by 9 bytes? Also - I wonder if the timestamp should be moved up and grouped with the other timestamps in the data structure now that there aren't matching flags as well?

Reply:
b8e4293

I moved the timestamp and it seemed to reduce the struct size to the expected number. Not sure why that happened but we've got what we want now.